2-mandatory/6-budgets.js

```diff
@@ -1,18 +1,24 @@
-/**
-Get Sum of People's Budget
-Create the function that takes an array with objects and returns the sum of people's budgets.
-For example:
-getBudgets([
+
+// Get Sum of People's Budget
+// Create the function that takes an array with objects and returns the sum of people's budgets.
+// For example:
+peopleArray = [
   { name: "John",  age: 21, budget: 29000 },
   { name: "Steve",  age: 32, budget: 32000 },
   { name: "Martin",  age: 16, budget: 1600 }
-])
-Should give return the answer of 62600.
-**/
+]
+// Should give return the answer of 62600.
+
 
 function getBudgets(peopleArray) {
+  let totalBudget = 0;
+  for (let budget of peopleArray){
+    totalBudget+=budget.budget
+  }
+  return totalBudget
```

Comment:
you can use also reduce here. Just as note. I like this solution better too :)

Reply:
than you changed it to 
peopleArray.reduce((acc, peopleArray)=>{ return acc + peopleArray.budget }, 0);